### PR TITLE
Adding more details to API group already registered error

### PR DESCRIPTION
**Pull request by nikhiljindal**
Also adding name of the group for which this error is raised.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
Ref https://github.com/kubernetes/kubernetes/issues/39494

**Comment by nikhiljindal** [comment]
friendly ping for @dchen1107

**Comment by nikhiljindal** [comment]
Rebased to move the change to staging/ dir.
cc @deads2k and @sttts

**Comment by nikhiljindal** [comment]
@k8s-bot cvm gce e2e test this

File: staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func (m *APIRegistrationManager) RegisterVersions(availableVersions []schema.Gro
 func (m *APIRegistrationManager) RegisterGroup(groupMeta apimachinery.GroupMeta) error {
 	groupName := groupMeta.GroupVersion.Group
 	if _, found := m.groupMetaMap[groupName]; found {
-		return fmt.Errorf("group %v is already registered", m.groupMetaMap)
+		return fmt.Errorf("group %s is already registered in groupsMap: %v", groupName, m.groupMetaMap)
```
Comment (sttts) [inline staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go]: nit: `%q` is even better

**Comment by sttts** [comment]
one nit, otherwise lgtm

File: staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go (hunk 1 above)
Comment (nikhiljindal) [inline staging/src/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go]: Done

**Comment by nikhiljindal** [comment]
Thanks @sttts for the review.
Fixed the nit as per comment.
Adding LGTM

**Comment by nikhiljindal** [comment]
Pushed a new rebased version to trigger all the tests that failed with "Error in queue".

**Pull request merged**